### PR TITLE
Revert ubuntu builds back to single stage

### PR DESCRIPTION
**Pull request by mbroz2**
Revert ubuntu builds back to single stage as requested by docker team for the official builds

### CODE REVIEW

File: releases/latest/beta/Dockerfile.ubuntu.openjdk8 [hunk 1]
```diff
@@ -41,7 +17,7 @@ LABEL org.opencontainers.image.authors="Leo Christy Jesuraj, Melissa Lee, Thomas
       org.opencontainers.image.revision="$LIBERTY_BUILD_LABEL" \
       org.opencontainers.image.description="This image contains the Open Liberty beta runtime with IBM Semeru Runtime Open Edition OpenJDK 8 with OpenJ9 and Ubuntu as the base OS.  For more information on this image please see https://github.com/OpenLiberty/ci.docker#building-an-application-image" \
       org.opencontainers.image.title="Open Liberty Beta" \
-      org.opencontainers.image.version="${LIBERTY_VERSION}"
```
Comment (tjwatson) [inline releases/latest/beta/Dockerfile.ubuntu.openjdk8:44]: Strange inconsistency.  Good to clean up.

**Review by tjwatson — APPROVED** [review]

File: releases/latest/beta/Dockerfile.ubuntu.openjdk8 (hunk 1 above)
Comment (mbroz2) [reply releases/latest/beta/Dockerfile.ubuntu.openjdk8:44]: ah, I thought I caught all of these

Comment (mbroz2) [reply releases/latest/beta/Dockerfile.ubuntu.openjdk8:44]: oh, I did.  This was the old version with the inconsistency

Comment (tjwatson) [reply releases/latest/beta/Dockerfile.ubuntu.openjdk8:44]: Sorry I was unclear on my comment.  I was saying it is good you cleaned that up.

**Review by leochr — COMMENTED** [review]
Ubuntu Dockerfiles of `latest/kernel-slim`, `23.0.0.6/kernel-slim` and `latest/beta` are missing the following:
- openssl installation
- `    && rm -rf /var/lib/apt/lists/* \`


Dockerfiles in `latest/full` and `23.0.0.6/full` are missing:
- `    && rm -rf /var/lib/apt/lists/* \`

Looks good otherwise. Thanks

**Comment by mbroz2** [comment]
@leochr 
I think technically we don't need to include openssl since it's already included in the base image, but we should certainly be consistent and since listing it will cause it to update the version if it's not already on the latest one, we might as well have it so I'll get it added.

Regarding `&& rm -rf /var/lib/apt/lists/* \`  I don't see it missing anywhere; it's included in all the `RUN apt-get update` sections after the `apt-get remove` commands.  Can you point it out in the review exactly where it's missing?

**Comment by leochr** [comment]
@mbroz2 Thanks for the update. I was looking for `&& rm -rf /var/lib/apt/lists/* \` n a different line and missed it. Sorry. Looks good. I'll go ahead and merge.

**Review by leochr — APPROVED** [review]

**Pull request merged**